common-test/pom.xml

```diff
@@ -40,6 +40,10 @@
             <artifactId>lombok</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.wiremock</groupId>
+            <artifactId>wiremock</artifactId>
```

Comment:
I think you are not using this dependency and it can be removed

Reply:
done